app/routes/frontpage/components/PublicFrontpage.css

```diff
@@ -2,25 +2,38 @@
 
 .container {
   display: grid;
-  grid-gap: 40px;
+  grid-gap: var(--spacing-xl);
   grid-template-columns: 1fr 1fr 1fr;
   grid-auto-rows: auto;
   grid-template-areas: 'welcome welcome login' 'events events hsp' 'article article readme' 'links links links';
+  padding: 0 var(--lego-default-padding);
 
   @media (--mobile-device) {
-    grid-gap: 20px;
     grid-template-columns: 1fr;
     grid-template-areas: 'welcome' 'login' 'events' 'hsp' 'article' 'readme' 'links';
+    padding: 0 var(--spacing-md);
   }
+
+  h1 {
+    margin-top: 0;
+  }
+
+  h3 {
+    margin-top: 0;
+    margin-left: 0;
+  }
+}
+
+.welcome i {
+  display: inline;
 }
 
 .login {
   height: fit-content;
 }
 
-.welcome,
 .links {
-  padding: 0 10px 10px;
+  padding: 0 var(--lego-default-padding) var(--lego-default-padding);
```

Comment:
Do we use this?

Reply:
That is the variable that's already used on the authenticated frontpage - so I only added it to use it on the public one as well for consistency.

That is however the only other place the variable is used, so I don't have any objections about yeeting it and rather just use `--spacing-xl`, though I don't really mind keeping it either..